src/dmtest/vdo/dedupe_tests.py

```diff
@@ -40,6 +40,49 @@ def t_dedupe75(fix):
     with standard_vdo(fix) as vdo:
         verify_dedupe(vdo, 0.75)
 
+def t_direct01_testBasic(fix):
```

Comment:
I think we should be designing and naming these test around the functionality, not the historic naming from the original VDO development repository. The fact that a given test was in the first (second, third) test file in our Perl tests doesn't really matter for dmtest. The name should give some hint as to what's being tested, if practical.

Reply:
I agree.  Any suggestions on general guidelines we should follow?

Reply:
What this test seems to add compared to the existing ones in that file is verifying data after restarting. The other tests verify the written data but they don't restart the device. So I'd suggest something about that for the name.

Reply:
I will bring this up in our meeting later today.  I was asking for suggestions on a consistent naming scheme to use.

Reply:
I think t_dedupeWithOffset would work for this.

Reply:
Renamed in latest push.  `t_dedupeWithOffsetAndRestart()`